src/SFA.DAS.LevyTransferMatching/Application/Commands/RejectApplication/RejectApplicationsHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MediatR;
+using SFA.DAS.LevyTransferMatching.Interfaces;
+using SFA.DAS.LevyTransferMatching.Models;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.LevyTransferMatching.InnerApi.LevyTransferMatching.Requests;
+
+namespace SFA.DAS.LevyTransferMatching.Application.Commands.RejectApplication
+{
+    public class RejectApplicationsHandler : IRequestHandler<RejectApplicationsCommand>
```

Comment:
The convention (perhaps not 100% adhered to) would make this `RejectApplications**Command**Handler`

Reply:
I have renames it now.